src/DynamoCoreWpf/Controls/StandardPanel.xaml

```diff
@@ -5,106 +5,114 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:controls="clr-namespace:Dynamo.Controls"
              xmlns:uicontrols="clr-namespace:Dynamo.UI.Controls"
+             xmlns:ui="clr-namespace:Dynamo.UI"
              mc:Ignorable="d"
              d:DesignHeight="300"
              d:DesignWidth="300"
              Background="Transparent"
              MinWidth="100">
     <UserControl.Resources>
-        <controls:BoolToVisibilityCollapsedConverter x:Key="BoolToVisibilityCollapsedConverter" />
-        <controls:IntToVisibilityConverter x:Key="IntToVisibilityConverter" />
-        <controls:NodeSearchElementVMToBoolConverter x:Key="NodeSearchElementVMToBoolConverter" />
-        <controls:BooleanToBrushConverter x:Key="BooleanToBrushConverter"
-                                          TrueBrush="#333333"
-                                          FalseBrush="#272727" />
+        <ResourceDictionary>
 
-        <Style TargetType="{x:Type ScrollViewer}">
-            <Setter Property="HorizontalScrollBarVisibility"
-                    Value="Hidden" />
-            <EventSetter Event="PreviewMouseWheel"
-                         Handler="OnShiftMouseWheel" />
-        </Style>
-        <Style x:Key="ListBoxItemStyle"
-               TargetType="{x:Type ListBoxItem}">
-            <Setter Property="BorderBrush"
-                    Value="Transparent" />
-            <Setter Property="BorderThickness"
-                    Value="1" />
-            <Setter Property="Height"
-                    Value="40" />
-            <Setter Property="ScrollViewer.CanContentScroll"
-                    Value="True" />
-            <Setter Property="FocusVisualStyle"
-                    Value="{x:Null}" />
-            <Setter Property="Visibility"
-                    Value="{Binding Path=Visibility, Converter={StaticResource BoolToVisibilityCollapsedConverter}}" />
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.SidebarGridDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
```

Comment:
I think you can remove `SidebarGridDictionaryUri`, and probably the other two also (need to try and see if it still works).